lib/mayaUsd/render/vp2RenderDelegate/render_delegate.h

```diff
@@ -155,10 +159,12 @@ class HdVP2RenderDelegate final : public HdRenderDelegate
         _resourceRegistry; //!< Shared and unused by VP2 resource registry
 
     std::unique_ptr<HdVP2RenderParam>
-            _renderParam; //!< Render param used to provided access to VP2 during prim synchronization
-    SdfPath _id;          //!< Render delegate IDs
+        _renderParam; //!< Render param used to provided access to VP2 during prim synchronization
+    SdfPath _id;      //!< Render delegate ID
```

Comment:
It is a change by clang-format.

Reply:
It doesn't seem to like comments after a member declaration. Wondering if there is a better configuration for it OR we should start simply putting documentation above the member.

Reply:
Indeed. I was thinking about having the doc above the member, but that would be another refactoring change. I am not familiar with clang-format, so I am not sure if there is a special configuration for this.